modules/nonce/commands/query.go

```diff
@@ -6,9 +6,10 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	lc "github.com/tendermint/light-client"
+	"github.com/tendermint/basecoin"
```

Comment:
I think we need a re-ordering of the imports... light-client and other tendermint together, then basecoin stuff...

I know they are all a bit mixed up after the merge.